iothub_client/src/iothub_module_client.c

```diff
@@ -68,6 +68,11 @@ IOTHUB_CLIENT_RESULT IoTHubModuleClient_SendReportedState(IOTHUB_MODULE_CLIENT_H
     return IoTHubClientCore_SendReportedState((IOTHUB_CLIENT_CORE_HANDLE)iotHubModuleClientHandle, reportedState, size, reportedStateCallback, userContextCallback);
 }
 
+IOTHUB_CLIENT_RESULT IoTHubModuleClient_GetDeviceTwinAsync(IOTHUB_MODULE_CLIENT_HANDLE iotHubModuleClientHandle, IOTHUB_CLIENT_DEVICE_TWIN_CALLBACK moduleTwinCallback, void* userContextCallback)
```

Comment:
>GetDeviceTwinAsync [](start = 40, length = 18)

For parity with other SDKs (and generally for readability), we should call it GetTwinAsync

Reply:
That's not a bad idea. It addresses both module and device client naming needs in a easy way. Will do was suggested.